content/en/network_performance_monitoring/installation.md

```diff
@@ -24,14 +24,19 @@ Supported **platforms** include:
 - SUSE 15+
 - Amazon AMI 2016.03+
 - Amazon Linux 2
+- [Windows 2016+][8] (in public beta)
 
-There is an exemption to the 4.4.0+ kernel requirement for [CentOS/RHEL 7.6+][2]. The [DNS Resolution][3] feature is not supported on CentOS/RHEL 7.6.
+**For Linux OS:** Data collection is done using eBPF, so Datadog minimally requires platforms that have an underlying Linux kernel versions of 4.4.0+.
+
+**For Windows OS:** Data collection is available in public beta for Windows versions 2016 or later.
```

Comment:
```suggestion
**For Windows OS:** Data collection is available in public beta for Windows versions 2016+.
```

Reply:
Same comment as below: The latest version of Windows is 2004 (the naming doesn't make sense to me, either), so "2016 or later" is intentional copy to avoid the confusion of "2016+".